bobtemplates/plone_addon/.isort.cfg

```diff
@@ -0,0 +1,6 @@
+[settings]
+force_alphabetical_sort = True
+force_single_line = True
+lines_after_imports = 2
+line_length = 200
+not_skip = __init__.py
```

Comment:
The fact is that there are already there. T_T generate a command line script in ``bin`` for it would be great too.